woocommerce-services.php

```diff
@@ -7,9 +7,9 @@
  * Author URI: https://woocommerce.com/
  * Text Domain: woocommerce-services
  * Domain Path: /i18n/languages/
- * Version: 1.19.0
+ * Version: 1.19.1
  * WC requires at least: 3.0.0
- * WC tested up to: 3.5.5
+ * WC tested up to: 3.6.1
```

Comment:
For the next times: We don't need to put `3.6.1`. In fact, that may give the impression that we don't support anything from `3.6.2` to `3.6.*`. WooCommerce core only checks major and minor version numbers, and assumes that a WooCommerce patch release can't possibly break anything: https://github.com/woocommerce/woocommerce/blob/393271013606a43f1fed9878054470289ff0cb7f/includes/admin/plugin-updates/class-wc-plugin-updates.php#L201-L213

So, putting `3.6` or even `3.6.*` would probably be marginally clearer.